soltesz-lab-to-nwb/treadmillbehaviordatainterface.py

```diff
@@ -0,0 +1,44 @@
+"""Authors: Cesar Echavarria."""
+from pynwb import NWBFile, TimeSeries
+from nwb_conversion_tools.basedatainterface import BaseDataInterface
+from nwb_conversion_tools.utils.json_schema import (
+    get_base_schema,
+    get_schema_from_hdmf_class,
+)
+from nwb_conversion_tools.utils.common_writer_tools import check_module
+from hdmf.backends.hdf5.h5_utils import H5DataIO
+
+import numpy as np
+
+
+class TreadmillBehaviorDataInterface(BaseDataInterface):
+    """Conversion class for running speed behavioral data."""
+
+    @classmethod
+    def get_source_schema(cls):
+        """Compile input schemas from each of the data interface classes."""
+        return dict(
+            required=["file_path"], properties=dict(file_path=dict(type="string"))
+        )
+
+    def run_conversion(
+        self,
+        nwbfile: NWBFile,
+        metadata: dict,
+        sampling_rate: float,
```

Comment:
Is sampling rate not defined in the .mat file?

Reply:
it's not. as far as I can tell, the .mat file only contains metadata relevant to the imaging data. The behavioral and ephys data are on a different sampling rate and neither file specifies the sampling rate so I think it's best to leave it as a user-defined parameter